docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -38,7 +38,7 @@ az extension list --output table
 ```output
 ExtensionType    Name                       Version
```

Comment:
Change the format of this table to include the `Name` column only so that the information which will frequently change can be elided.